data.tf

```diff
@@ -0,0 +1,22 @@
+data "aws_vpc" "vpc" {
+  id = var.vpc_id
+}
+
+data "aws_availability_zones" "azs" {
```

Comment:
Plz check if this block is used or not in the code bcoz I have not found a reference.